src/SetSignalHandling.cpp

```diff
@@ -27,28 +27,34 @@
 namespace LvArray
 {
 
+typedef void ( *handle_type )( int );
+static handle_type initialHandler[128] = { nullptr };
```

Comment:
I would make this a `std::map< int, handle_type >` from signal to handler so that you don't have to worry about the ordering.

Reply:
So much better that way.